influxdb3_write/src/write_buffer/validator.rs

```diff
@@ -637,7 +628,13 @@ fn convert_v3_parsed_line(
     // Add the row into the correct chunk in the table
     let chunk_time = gen1_duration.chunk_time_for_timestamp(Timestamp::new(time_value_nanos));
     let table_name: Arc<str> = line.series.measurement.to_string().into();
-    let table_chunks = table_chunk_map.entry(Arc::clone(&table_name)).or_default();
+    let table_id = db_schema
+        .get_table(&table_name)
+        .map(|t| t.table_id)
+        .unwrap_or_else(|| TableId::from(0));
```

Comment:
Should this take the next available table ID, instead of using 0?

Reply:
(the same would be said for the other parsing function)

Reply:
Hmmmmm I was unsure. It's validating not creating a new table as far as I understood.

Reply:
Ah, I see, the code in validator is a bit confusing right now. For example, it used to update the in-memory catalog schema using a `Cow` to check that it had changes which is why this is here: https://github.com/influxdata/influxdb/blob/9c71b3ce251f32cf8f23db0a4f09873e04686c1a/influxdb3_write/src/write_buffer/validator.rs#L514-L521

But it doesn't look like it uses the cow anymore. It does still update the catalog in its state here, though, by applying the catalog batch: https://github.com/influxdata/influxdb/blob/9c71b3ce251f32cf8f23db0a4f09873e04686c1a/influxdb3_write/src/write_buffer/validator.rs#L182-L196

And I believe that will result in the table being created, so I think it should probably be using the next ID instead of 0. Perhaps a test would help suss that out.